BUILD.gn

```diff
@@ -35,7 +35,7 @@ import("//src/crypto/crypto.gni")
 if (current_toolchain != "${dir_pw_toolchain}/default:default") {
   declare_args() {
     chip_enable_python_modules =
-        (current_os == "mac" || current_os == "linux") &&
+        (current_os == "mac" || current_os == "linux" || current_os == "android") &&
         (host_cpu == "x64" || host_cpu == "arm64" || host_cpu == "arm")
     enable_pylib = false
```

Comment:
I don't know the gn build system well enough to evaluate whether this is correct.  Is it true that setting `enable_pylib` false enables `chip_enable_schema_check`?

Also, is there a way to override this parameter for the build?  If so, would it be appropriate to do so?  I am only asking the question.  If this is about uniformity and providing a sensible default, I'm sure it makes sense.

Reply:
The logic seems to say "enable python modules if we compile for android" ... is this really what we want?

Reply:
I'm sorry It is different file changes during the patch migration. we want to change src/app/BUILD.gn not root.

Reply:
We have changed commit again. Please review this patch again @andy31415